dts/msm8226/msm8926-huawei-g6-l11.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	model = "Qualcomm MSM 8926 CDP G6-L11";
```

Comment:
This should be the real device name (e.g. `Huawei Ascend ...`).

Reply:
Indeed, acknowledged.